avatars/avatars.js

```diff
@@ -1635,6 +1643,9 @@ class Avatar {
       const currentSpeed = localVector.set(this.velocity.x, 0, this.velocity.z).length();
       const angle = this.getAngle();
       const timeSeconds = now/1000;
+
+      this.speedValue = lerpFloat(this.speedValue,currentSpeed,0.1);
```

Comment:
This looks dependent on the frame time.

Reply:
I should have use fimeDiff in this calculation